solutions/python-igraph/visualize_cliques/visualize_cliques_with_edges.py

```diff
@@ -0,0 +1,27 @@
+import igraph as ig
+import matplotlib.pyplot as plt
+
+g = ig.Graph.Famous('Zachary')
+g.vs['label'] = [i + 1 for i in range(g.vcount())]
+g.vs['size'] = 0
```

Comment:
Instead of setting this before the `for` loop and after each plotting statement, I would simply do this once before the plotting.